windows-driver-docs-pr/stream/360-camera-video-capture.md

```diff
@@ -114,9 +114,9 @@ AddReg=ContosoSampleDeviceMFT_COM.AddReg
 ;-----------------------------------------------------------------------------------
 
 [ContosoSampleDeviceMFT_COM.AddReg]
-HKCR,CLSID\%SampleDeviceMFT.CLSID%,,,%SampleDeviceMFT.FriendlyName%
-HKCR,CLSID\%SampleDeviceMFT.CLSID%InProcServer32\,,,%%SystemRoot%%\ContosoSampleDeviceMFT.dll
-HKCR,CLSID\%SampleDeviceMFT.CLSID%InProcServer32\,ThreadingModel,,"Both"
+HKR,Classes\CLSID\%SampleDeviceMFT.CLSID%,,,%SampleDeviceMFT.FriendlyName%
```

Comment:
You should update the Models section to only be applicable on OS versions that support the MFT COM class being registered with device relative COM registrations.